Add in documentation on APIv4 Options (#706)
civicrm/civicrm-dev-docs: docs/api/v4/options.md

# APIv4 Options

> There are many API Options accepted by the CiviCRM API. These options allow the developer to add in more parameters to the resulting Query that is run against the database. E.g. Limit, Sort. You can explore these options using the the [API Explorer](/api/index.md#api-explorer) Unlike in APIv3 setting the options is not contained within an array of options in the parameters array.

Reviewer: This is worded a little unclearly towards the end.

> ## Orderby

Reviewer: ```suggestion
## OrderBy
```

> There are many API Options accepted by the CiviCRM API. These options allow the developer to add in more parameters to the resulting Query that is run against the database. E.g. Limit, Sort. You can explore these options using the the [API Explorer](/api/index.md#api-explorer) In APIv3 options were set as keys of an array called options within the parameters array. In APIv4 tthey are just passed in as another parameter not in a nested array.

Reviewer 2: Some copy-edits:

```suggestion
The CiviCRM API accepts many *options*. These options allow the developer to refine the behavior of the API call -- for example, setting a *limit* on the number of records or the *order* of the returned records.

Every API may support distinct options. In this page, we'll examine a few of the most common options in APIv4. For the most detailed information about available options, consult the [APIv4 Explorer](/api/index.md#api-explorer).
```

Substantively, this suggestion does strike the comparison between APIv3/APIv4. The comparison is a good idea, but I'm wondering if maybe it should be a separate document. For someone reading *today*, the comparison is probably more important than the reference-material for APIv4. OTOH, if someone is reading this in, say, 12 months -- then the comparisons would be by-the-by.

Maybe we need a document like "APIv4 for People Who Know APIv3". (Very rough cut: https://gist.github.com/totten/cd2e62e32c84fa6e1b0a1021ffe0a252) @seamuslee001 @colemanw @MikeyMJCO - I wonder if that kind of content belongs in the Dev Guide - or perhaps maybe more appropriate to a blog?

Reviewer: The suggested edit is good. I think the content you describe would be more appropriate as a blog. We can link to it from the dev guide if appropriate.

-   **Action**: get
-   **Type**: string
-   **Default** id ASC

> Determine what the orer of the records should be. This was either done with `options.sort` or `options.sequential` in APIv3

Reviewer: ```suggestion
Determine what the order of the records should be. This was either done with `options.sort` or `options.sequential` in APIv3
```
